tests/system/model_monitoring/test_app.py

```diff
@@ -110,7 +112,7 @@ def custom_setup(cls, project_name: str) -> None:
             project_name, "./", allow_cross_project=True
         )
         project.set_model_monitoring_credentials(
-            stream_path=mlrun.mlconf.model_endpoint_monitoring.stream_connection,
+            stream_path=os.getenv("MLRUN_MODEL_ENDPOINT_MONITORING__STREAM_CONNECTION"),
```

Comment:
I think we should change it to use profile as it the main flow now - relevant to all the MM tests.

Reply:
I definitely agree and intend to do it gradually after we support the TSDB profile.
Opened [ML-9020](https://iguazio.atlassian.net/browse/ML-9020).